app/Console/Commands/LocaleCheck.php

```diff
@@ -149,33 +150,12 @@ public function everything($locale, $heads)
         return $entries;
     }
 
-    public function files($locale, $path = null)
+    public function files($locale)
     {
-        if ($path === null) {
-            $path = $this->basePath($locale);
-        }
-
-        $entries = scandir($path);
         $files = [];
 
-        foreach ($entries as $entry) {
-            if ($entry === '.' || $entry === '..') {
-                continue;
-            }
-
-            $entryPath = $path.'/'.$entry;
-
-            if (is_link($entryPath)) {
-                continue;
-            }
-
-            if (is_dir($entryPath)) {
-                $files = array_merge($files, $this->files($locale, $entryPath));
-            }
-
-            if (is_file($entryPath) && ends_with($entryPath, '.php')) {
-                $files[] = $entryPath;
-            }
+        foreach (File::allFiles($this->basePath($locale)) as $entry) {
```

Comment:
can just return `File::allFiles` directly and `->getPathname()` in `heads()` or some of those other `SplFileInfo` methods